### ISSUE TITLE
Blazor Add-In stopped working in .NET 5.0

### ISSUE THREAD

**Issue by LeonGorbaty1** [manuelsidler/blazor-office-addin#7]
@manuelsidler hi again,

The Blazor (server) was working fine until I changed "Target framework" from .NET Core 3.1 to .NET 5.0 (which is RC1).
The page still loads in the add-in, but no event handlers are working, and I noticed that OnAfterRender (and OnAfterRenderAsync) are not being called any more ... can you reproduce this ... any idea?

THANK YOU, Leon

**Comment by manuelsidler** [comment]
Hi Leon

I did not try my sample with .NET 5 yet. I'll let you know.

**Comment by manuelsidler** [comment]
Yep, seems to be broken. I may have some time during the holiday season to get it working again.
